### PR TITLE
Fake the job descriptor instead of using the pod annotation

### PR DESCRIPTION
**Pull request by solarkennedy**
We would like to get rid of the job descriptor from our pods.

It doesn't look like this code cares too much about the actual contents,
so I reproduce the few things it cared about from the pod spec directly.

### CODE REVIEW

File: titus_isolate/model/pod_utils.py [hunk 1]
```diff
@@ -93,9 +94,41 @@ def get_entrypoint(job_descriptor: dict) -> str:
 
 
 def get_job_type(job_descriptor: dict) -> str:
+    # For whatever reason, on v0 pods for "job_type" we returned the image?
```
Comment (gabrielhartmann) [inline titus_isolate/model/pod_utils.py:97]: You're right.  This is a bug.  It should be BATCH or SERVICE

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (gabrielhartmann) [inline .vscode/settings.json:1]: Can we please gitignore IDE specific files?

File: test-requirements.txt [hunk 3]
```diff
@@ -10,3 +10,4 @@ netflix-spectator-py
 kubernetes
 grpcio
 grpcio-tools
+scs==2.0.1
```
Comment (gabrielhartmann) [inline test-requirements.txt:13]: What's this for?

**Review by gabrielhartmann — APPROVED** [review]

File: test-requirements.txt (hunk 3 above)
Comment (solarkennedy) [reply test-requirements.txt:13]: For test, I found that pinning this version of scs was the only way I could get a working venv + pandas.
My best guess is that something broke with latest scs.
Who knows though, could be python, my laptop, macos. I'll undo.

**Pull request merged**